x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/details_step/details_step_form.tsx

```diff
@@ -240,10 +239,7 @@ export const DetailsStepForm: FC<CreateAnalyticsStepProps> = ({
                   }
                 )}
                 <br />
-                <EuiLink
-                  href={`${ELASTIC_WEBSITE_URL}guide/en/elasticsearch/reference/${DOC_LINK_VERSION}/indices-create-index.html#indices-create-index`}
-                  target="_blank"
-                >
+                <EuiLink href={createIndexLink} target="_blank">
```

Comment:
Just wondering, where we use an `EuiLink` in this way, should the text end with a ` .` ? Some places we end with a `.` and some we don't. 

![image](https://user-images.githubusercontent.com/7405507/103893559-63fa1a00-50e5-11eb-946e-b25655d296f2.png)

I prefer without the `.`

![image](https://user-images.githubusercontent.com/7405507/103893674-9277f500-50e5-11eb-828a-c20debe338ae.png)

Reply:
There's an example of "Learn more" with an ending period in https://elastic.github.io/eui/#/guidelines/writing and a quick chat with @gchaps indicates it's preferred in general.  I will open a separate PR to get consistency on that.